docs/guides/genomics-platform/analyzing-data/sequencerr.md

```diff
@@ -80,11 +69,60 @@ You can navigate to the **SequencErr** application [here](https://platform.stjud
     Set this to prevent large count files from being generated
     Default: true
 
-**Provide Input Parameters and Run the Analysis**
-![](../../../files/guides/tools/sequencerr/SequencErr_steps_7a-10.png)
+## Running the Analysis
+
+**SequencErr** application can be accessed from [https://platform.stjude.cloud/workflows/sequencerr](https://platform.stjude.cloud/workflows/sequencerr)
+
+!!! note
+    This tool is intended free-of-charge for non-profit usages. 
+    Please contact [Dr. Xiaotu Ma](mailto:Xiaotu.Ma@stjude.org) for for-profit usages and modifications
```

Comment:
It's free for commercial accounts to access this as well. Not saying that this is a bad thing to put in here, but I doubt anybody is going to contact you about it.

Reply:
We don't have control over it. It is a general practice to advise users.